django-spectre-css/templatetags/spectre_css.py

```diff
@@ -19,14 +19,19 @@ def is_select(field):
     return isinstance(field.field.widget, Select)
 
 
+@register.filter
+def is_multi_field(field: BoundField):
+    return "fields" in field.field.__dict__
```

Comment:
Is there no cleaner way to check for a MultiField? E.g., something like issubclass or `isinstance`?